### PR TITLE
Makes various improvements to the docs

### PR DESCRIPTION
**Pull request by tanertopal**
#### What does this implement/fix? Explain your changes.

Fixing and improving docs.

### CODE REVIEW

File: doc/source/contributor-setup.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ Install a ``flwr`` pre-release from PyPI:
 - ``pip install -U --pre flwr`` (without extras)
 - ``pip install -U --pre flwr[simulation]`` (with extras)
 
-Python packages can be installed from git repositories. Use the following command to install the latest version of Flower directly from GitHub::
+Python packages can be installed from git repositories. Use the following command to install the latest version of Flower directly from GitHub.
```
Comment (danieljanes) [inline doc/source/contributor-setup.rst:35]:
```suggestion
Python packages can be installed from git repositories. Use one of the following commands to install the Flower directly from GitHub.
```

File: doc/source/_static/custom.css [hunk 2]
```diff
@@ -1,5 +1,7 @@
-@import url(https://fonts.googleapis.com/css?family=Inter:400,600);
-
 html, body{
-    font-family: Inter,BlinkMacSystemFont,-apple-system,Segoe UI,Roboto,Oxygen,Ubuntu,Cantarell,Fira Sans,Droid Sans,Helvetica Neue,Helvetica,Arial,sans-serif;
+    font-family: Montserrat,ui-sans-serif,system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol,Noto Color Emoji;
```
Comment (danieljanes) [inline doc/source/_static/custom.css:2]: Doesn't this change the font?

File: src/py/flwr/server/strategy/fedadagrad.py [hunk 3]
```diff
@@ -85,12 +85,8 @@ def __init__(
                 during validation. Defaults to 2.
             min_available_clients (int, optional): Minimum number of total
                 clients in the system. Defaults to 2.
-            evaluate_fn : Optional[
-                Callable[
-                    [int, NDArrays, Dict[str, Scalar]],
-                    Optional[Tuple[float, Dict[str, Scalar]]]
-                ]
-            ]: Function used for validation. Defaults to None.
+            evaluate_fn : Optional[Callable[[int, NDArrays, Dict[str, Scalar]], Optional[Tuple[float, Dict[str, Scalar]]]]]
```
Comment (danieljanes) [inline src/py/flwr/server/strategy/fedadagrad.py:88]: Why do we need this?

**Review by danieljanes — CHANGES_REQUESTED** [review]

File: src/py/flwr/server/strategy/fedadagrad.py (hunk 3 above)
Comment (tanertopal) [reply src/py/flwr/server/strategy/fedadagrad.py:88]: Current rendering is broken see here:
Screenshot 2022-10-05 at 13 36 31

Comment (tanertopal) [reply src/py/flwr/server/strategy/fedadagrad.py:88]: vs.
Screenshot 2022-10-05 at 13 42 21

File: doc/source/_static/custom.css (hunk 2 above)
Comment (danieljanes) [reply doc/source/_static/custom.css:2]: @tanertopal that's nice

File: src/py/flwr/server/strategy/fedadagrad.py (hunk 3 above)
Comment (danieljanes) [reply src/py/flwr/server/strategy/fedadagrad.py:88]: @tanertopal before we do this, can we format the docstrings in question using NumPy style (the format that we try to use everywhere)?

**Review by danieljanes — CHANGES_REQUESTED** [review]

File: src/py/flwr/server/strategy/fedadagrad.py [hunk 4]
```diff
@@ -37,15 +37,15 @@
 from .fedopt import FedOpt
 
 
-# pylint: disable=too-many-locals
+# flake8: noqa: E501
```
Comment (danieljanes) [inline src/py/flwr/server/strategy/fedadagrad.py:40]: This should be enough, right? That's how we have it in other parts of the codebase
```suggestion
# noqa: E501
```

File: src/py/flwr/server/strategy/fedavg.py [hunk 5]
```diff
@@ -104,7 +99,7 @@ def __init__(
             Whether or not accept rounds containing failures. Defaults to True.
         initial_parameters : Parameters, optional
             Initial global model parameters.
-        fit_metrics_aggregation_fn: Optional[MetricsAggregationFn]
+        fit_metrics_aggregation_fn : Optional[MetricsAggregationFn]
             Metrics aggregation function, optional.
         evaluate_metrics_aggregation_fn: Optional[MetricsAggregationFn]
```
Comment (danieljanes) [inline src/py/flwr/server/strategy/fedavg.py:104]:
```suggestion
        evaluate_metrics_aggregation_fn : Optional[MetricsAggregationFn]
```

**Review by danieljanes — CHANGES_REQUESTED** [review]

File: src/py/flwr/server/strategy/fedadagrad.py (hunk 4 above)
Comment (tanertopal) [reply src/py/flwr/server/strategy/fedadagrad.py:40]: @danieljanes only if it's for a specific line. As it's in the doc string, I can't really put it into that line.

**Review by danieljanes — APPROVED** [review]

**Pull request merged**